src/app/about/page.tsx

```diff
@@ -0,0 +1,13 @@
+import AboutPage from "../../components/AboutPage/AboutPage";
```

Comment:
Why do you import it like that?
shouldn't be path be like:
import AboutPage from "../../components/AboutPage";?